### PR TITLE
Implemented test cases NMS_FT_86, 87, 88

### PR DESCRIPTION
**Pull request by kanchannms**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by koshalt** [comment]
ok to test

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 1]
```diff
@@ -783,4 +785,34 @@ public void verifyFT77()
                 expectedJsonPattern, ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
+    @Ignore
```
Comment (koshalt) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: Couple of questions - why is this marked ignore? Do we already have a jira ticket for the failures? Can you add it here, if that's true.

Also, it is usually good practice to have a test case per scenario to make that they are aren't cascading failures and it is easy from a failure to pinpoint a particular scenario than mixing them up

**Comment by nmsyogesh** [comment]
A JIRA issue has been raised for tracking this at https://applab.atlassian.net/browse/NMS-186.

**Comment by frankhuster** [comment]
please merge

**Comment by srivastavaAbhi** [comment]
Tracked with branch NMS.FT.KILKARICONTROLLERIT.
So closing this request.

**Pull request closed without merging**